src/style.css

```diff
@@ -0,0 +1,78 @@
+body {
+  margin: 0;
+}
+
+.grid {
+  display: grid;
+  grid-template-areas: "grid_a_1 grid_a_2 grid_a_3"
+    "grid_a_4 grid_a_5 grid_a_6";
+  grid-gap: 0;
+  padding: 0;
+}
+
+.grid__block {
+  min-width: 300px;
+  min-height: 300px;
+}
+
+.grid__block-text {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: white;
+  position: relative;
+  float: left;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.grid__block-first {
+  background-color: hsl(0, 100%, 50%);
+  grid-area: grid_a_1;
+}
+
+.grid__block-second {
+  background-color: hsl(0, 100%, 40%);
+  grid-area: grid_a_2;
+}
+
+.grid__block-third {
+  background-color: hsl(0, 100%, 30%);
+  grid-area: grid_a_3;
+}
+
+.grid__block-fourth {
+  background-color: hsl(0, 100%, 20%);
+  grid-area: grid_a_4;
+}
+
+.grid__block-fifth {
+  background-color: hsl(0, 100%, 10%);
+  grid-area: grid_a_5;
+}
+
+.grid__block-sixth {
+  background-color: hsl(0, 100%, 0%);
+  grid-area: grid_a_6;
+}
+
+@media (max-width: 900px) {
+  .grid {
+    display: grid;
+    grid-template-areas: "grid_a_1 grid_a_2"
+      "grid_a_3 grid_a_4"
+      "grid_a_5 grid_a_6";
```

Comment:
I think if you are making this via areas will be better if you will change grid-area in element instead of changing whole grid

Reply:
@earthhuman41 ,    sorry, can't understand what to do.. 😑

Reply:
@Denchepornyuk , please advice. I'm stucked